### ISSUE TITLE
btcctl not installed after following lnd install instructions

### ISSUE THREAD

**Issue by charlespax** [lightningnetwork/lnd#2742]
### Background

I am following the installation instructions at https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md. A part of the instructions states the following:

> Additionally, you can monitor btcd's logs to track its syncing progress in real time.
> 
> You can test your btcd node's connectivity using the getpeerinfo command:
> 
> btcctl --testnet --rpcuser=REPLACEME --rpcpass=REPLACEME getpeerinfo | more

I've followed the instructions and I'm syncing right now using `btcd --txindex`. Unfortunately, `btcctl` does not seem to be installed. The instructions do not state how to install `btcctl` and do not reference external instructions on installing `btcctl`. I believe the instructions should either include information on how to install `btcctl` or do not reference this command.

It may also be that I did not follow an instruction correctly.

### Your environment

* version of `lnd`: `lnd version 0.5.2-99-beta commit=v0.5.1-beta-816-g3e11d...c02413f`
* which operating system (`uname -a` on *Nix): `Linux 4.18.0-15-generic #16-Ubuntu SMP`
* version of `btcd`, `bitcoind`, or other backend: `btcd`
* any other relevant environment details

### Steps to reproduce

I am following the lnd install instructions and writing a helper script for myself. I have installed the system by running my script, which is located at https://github.com/charlespax/nodalizer

### Expected behaviour

I expect to be able to execute commands in the instructions.

### Actual behaviour

A command in the instructions does not execute.

**Comment by molxyz** [comment]
@charlespax Do you have at least go v1.11 installed? 
You can follow this guide  `https://github.com/btcsuite/btcd#linuxbsdmacosxposix---build-from-source` to build btcd . 
I think if you run this command it will install btcctl for you: 
```
$ cd $GOPATH/src/github.com/btcsuite/btcd
$ GO111MODULE=on go install -v . ./cmd/...
```

**Comment by Roasbeef** [comment]
You need to install `btcctl` separately.

**Comment by charlespax** [comment]
I went to that Go installation page. I didn't follow their Go installatino instructions as I already have it installed.

I then executed the btcd installation commands on that page, which you also have in your comment.
'''
$ go version
go version go1.11.5 linux/amd64
$ go env GOROOT GOPATH
/usr/local/go
/home/charles/gocode
$ cd $GOPATH/src/github.com/btcsuite/btcd
$ GO111MODULE=on go install -v . ./cmd/...
go: unknown environment setting GO111MODULE=ON
'''
Go then downloads and installs a bunch of stuff. `btcctl` is now installed on my system. Thank you :-)

**Comment by charlespax** [comment]
I made a pull request to modify the instructions (https://github.com/lightningnetwork/lnd/pull/2743). An alternative would be to add @molxyz's link to INSTALL.md. Or at least mention that btcctl` must be installed separately. Thoughts?

**Comment by molxyz** [comment]
@charlespax Did you use `make btcd` at first and that was why it didn't install `btcctl` for you? 

@Roasbeef  `make btcd` has never worked for me.  I am thinking maybe the doc should be changed by just pointing users to the install guide on  `https://github.com/btcsuite/btcd`; and put `make btcd` as an alternative on that site if you still want to keep it ?  

In the past I had to install `btcctl` separately but so far I've found the btcsuite guide does it by just running that command.

**Comment by charlespax** [comment]
@molxyz Yes, I installed `btcd` with the following commands.
```
cd $GOPATH/src/github.com/lightningnetwork/lnd
make btcd
```
After executing the above commands `btcd` was installed, but not `btcctl`.

**Comment by cfromknecht** [comment]
@charlespax @molxyz heh, just checked the makefile and it doesn't actually install btcd, it only fetches the proper version, see #2745

**Comment by charlespax** [comment]
I was just looking at that. Relevant line in the Makfile:
```
83  btcd:                                                                                                                            
84     @$(call print, "Installing btcd.")
85     GO111MODULE=on go get -v github.com/btcsuite/btcd/@$(BTCD_COMMIT)
```
Should I close my pull request regarding the instructions change or wait until #2745 has been merged?

**Comment by cfromknecht** [comment]
would say wait for some feedback on #2745 as that is the intended behavior of the directive, as the docs indicate

**Comment by Roasbeef** [comment]
Fixed by #2745.
